New SIGNL4 alerting article (#3145)
SumoLogic/sumologic-documentation: docs/alerts/webhook-connections/signl4.md

---
id: signl4
title: Webhook Connection for SIGNL4
sidebar_label: SIGNL4
description: Send alerts to SIGNL4 from scheduled searches.
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('https://raw.githubusercontent.com/signl4/signl4-logo/main/signl4.png')} alt="Thumbnail icon" width="50"/>

> [Webhook connections](/docs/alerts/webhook-connections/set-up-webhook-connections) rely on HTTP endpoints that tell Sumo Logic where to send data. You can set up any number of connections. 

Reviewer: ```suggestion
You can set up [webhook connections](/docs/alerts/webhook-connections/set-up-webhook-connections) rely on HTTP endpoints that tell Sumo Logic where to send data.
```

## Sumo Logic to SIGNL4 Integration

Sumo Logic alerts can send webhook alerts to SIGNL4, a mobile alerting and incident management solution that determines the right people to alert based on your on-call schedule, severity, and topic.

To add a Sumo Logic integration in SIGNL4, do the following:

1. Go to the [SIGNL4 Integration Hub](https://account.signl4.com/manage/IntegrationHub?tabName=installedApps).
1. Select an existing webhook integration or create a new one for the Sumo Logic integration.

> 1. In the integration tile copy the URL including integration or team secret.

Reviewer: ```suggestion
1. In the integration tile, copy the URL including integration or team secret.
```

> You need the webhook URL for the configuration in Sumo Logic.

Reviewer: ```suggestion
You'll need the webhook URL for Sumo Logic configuration, which you'll do in the next section.
```

## Configuration in Sumo Logic

In Sumo Logic, scheduled searches send alerts to other tools via webhook connections. To send alerts from Sumo Logic to SIGNL4:

1. [Create a Webhook Connection](#create-a-webhook-connection).
1. Once you set up the webhook connection, you'll have the option to use it in a [Scheduled Search](/docs/alerts/webhook-connections/schedule-searches-webhook-connections)  or [Monitor](/docs/alerts/monitors).

## Create a Webhook Connection

:::note
You need the **Manage connections** [role capability](/docs/manage/users-roles/roles/role-capabilities) to create webhook connections.
:::

This section demonstrates how to create a webhook connection from Sumo Logic to SIGNL4.

To create a webhook:

1. In Sumo Logic, go to **Manage Data > Alerts > Connections**.
1. Click **+ Add** and choose **Webhook** as connection type.
1. For the name, enter **Sumo Logic SIGNL4** and give an optional description to the connection.
1. Paste the SIGNL4 webhook URL (from the step above) into the **URL** field.
1. Enter the following content in the **Alert Payload** field:

>     ```

Reviewer: ```suggestion
    ```json
```

>     ```json

Reviewer: ```suggestion
    ```
```
